packages/ovm/test/contracts/execution-manager.call-opcodes.spec.ts

```diff
@@ -324,33 +323,41 @@ describe('Execution Manager -- Call opcodes', () => {
 
   describe('ovmSTATICCALL', async () => {
     it('properly executes ovmSTATICCALL to SLOAD', async () => {
-      const result = await executeOVMCall(executionManager, 'executeCall', [
-        0,
-        0,
-        addressToBytes32Address(callContractAddress),
-        methodIds.makeStaticCall,
-        addressToBytes32Address(callContract2Address),
-        methodIds.staticFriendlySLOAD,
-        sloadKey,
-      ])
+      const result = await executeOVMCall(
+        executionManager,
+        'executeTransactionRaw',
+        [
+          getCurrentTime(),
+          0,
+          addressToBytes32Address(callContractAddress),
+          methodIds.makeStaticCall,
+          addressToBytes32Address(callContract2Address),
+          methodIds.staticFriendlySLOAD,
+          sloadKey,
+        ]
+      )
 
       log.debug(`Result: [${result}]`)
 
       remove0x(result).should.equal(unpopultedSLOADResult, 'Result mismatch!')
     })
 
     it('properly executes nested ovmSTATICCALL to SLOAD', async () => {
-      const result = await executeOVMCall(executionManager, 'executeCall', [
-        0,
-        0,
-        addressToBytes32Address(callContractAddress),
-        methodIds.makeStaticCall,
-        addressToBytes32Address(callContract2Address),
-        methodIds.makeStaticCall,
-        addressToBytes32Address(callContract2Address),
-        methodIds.staticFriendlySLOAD,
-        sloadKey,
-      ])
+      const result = await executeOVMCall(
+        executionManager,
+        'executeTransactionRaw',
```

Comment:
`executeTransactionRaw` is used as a literal a number of times here. Worth it to create a const? Could be useful to have it in `helpers` or something since it's frequently used across different test files, although adhering to using the const will be a separate battle.

Reply:
Good call but I'm going to leave this one for now so we can do a good sweep of all strings to constants in it's own PR 🚀